common/djangoapps/third_party_auth/api/urls.py

```diff
@@ -14,6 +14,7 @@
         UserView.as_view(),
```

Comment:
I vaguely recall even wrapping this deprecated url pattern behind a feature toggle/django setting, but that may have been part of conversations with my team.  @doctoryes @macdiesel what are your thoughts about making this togglable?

Reply:
Oh, well we can't do that of course because it impacts an active endpoint.  I'll just drop this concern for now.